### ISSUE TITLE
How to trash environment

### ISSUE THREAD

**Issue by wbvtler** [codercom/code-server#211]
## Description
I tried updating from Ubuntu 16.04 to 18.04 and it did not work as expected, which then caused a number of additional issues.

Is there a way to simply trash this environment and start over?

**Comment by sr229** [comment]
If this is `ide.coder.com`, then sadly you tried to run a distro upgrade in a Docker container, which does NOT work.

**Comment by wbvtler** [comment]
Yeah, I figured that out. Now I just want to essentially do a hard-reset. I hadn't even started coding, so I'm not worried about losing anything.

**Comment by sr229** [comment]
@wbvtler Help -> Restart Container

**Comment by sr229** [comment]
If that doesn't work, contact Support.

**Comment by wbvtler** [comment]
@sr229 I want to fully trash all of my changes to the container and start over from the original image, doing a restart from within the IDE only turns it off and on again.

**Comment by coadler** [comment]
You'll have to contact support because we removed the reset container button as the behavior of it changed when we released `code-server`.

Gonna close this issue since it's for our website and not the open source part. We have https://github.com/codercom/bugs/issues and https://github.com/codercom/features/issues as issue trackers for the coder website.
